plugins/woocommerce-admin/client/core-profiler/pages/BusinessInfo.tsx

```diff
@@ -1,34 +1,340 @@
+						onChange={ ( value ) => {
+							setStoreName( value );
+						} }
+						value={ storeName }
+						label={
+							<>
+								{ __(
+									'Give your store a name',
+									'woocommerce'
+								) }
+							</>
+						}
+						placeholder={ __(
+							'Ex. My awesome store',
+							'woocommerce'
+						) }
+					/>
+					<p className="woocommerce-profiler-question-subtext">
+						{ __(
+							"Don't worry — you can always change it later!",
+							'woocommerce'
+						) }
+					</p>
+					<p className="woocommerce-profiler-question-label">
+						{ selectIndustryQuestionLabel }
+					</p>
+					<SelectControl
+						className="woocommerce-profiler-select-control__industry"
+						instanceId={ 1 }
+						placeholder={ __(
+							'Select an industry',
+							'woocommerce'
+						) }
+						label={ __( 'Select an industry', 'woocommerce' ) }
+						options={ industryChoices }
+						excludeSelectedOptions={ false }
+						help={ <Icon icon={ chevronDown } /> }
+						onChange={ (
+							results: Array< typeof industryChoices[ number ] >
+						) => {
+							if ( results.length ) {
+								setIndustry( results[ 0 ] );
+							}
+						} }
+						selected={ industry ? [ industry ] : [] }
+						showAllOnFocus
+						isSearchable
+					/>
+					<p className="woocommerce-profiler-question-label">
+						{ __( 'Where is your store located?', 'woocommerce' ) }
+						<span className="woocommerce-profiler-question-required">
+							{ '*' }
+						</span>
+					</p>
+					<SelectControl
+						className="woocommerce-profiler-select-control__country"
+						instanceId={ 2 }
+						placeholder={ selectCountryLabel }
+						label={
+							storeCountry.key === '' ? selectCountryLabel : ''
+						}
+						getSearchExpression={ ( query: string ) => {
+							return new RegExp(
+								'(^' + query + '| — (' + query + '))',
+								'i'
+							);
+						} }
+						options={ countries }
+						excludeSelectedOptions={ false }
+						help={ <Icon icon={ chevronDown } /> }
+						onChange={ ( results: Array< CountryStateOption > ) => {
+							if ( results.length ) {
+								setStoreCountry( results[ 0 ] );
+							}
+						} }
+						selected={ storeCountry ? [ storeCountry ] : [] }
+						showAllOnFocus
+						isSearchable
+					/>
+					{ geolocationOverruled && ! dismissedGeolocationNotice && (
+						<Notice
+							className="woocommerce-profiler-geolocation-notice"
+							onRemove={ () =>
+								setDismissedGeolocationNotice( true )
+							}
+							status="warning"
+						>
+							<p>
+								{ createInterpolateElement(
+									__(
+										// translators: first tag is filled with the country name detected by geolocation, second tag is the country name selected by the user
+										"It looks like you're located in <geolocatedCountry></geolocatedCountry>. Are you sure you want to create a store in <selectedCountry></selectedCountry>?",
+										'woocommerce'
+									),
+									{
+										geolocatedCountry: (
```

Comment:
I think we need to make this clickable so that the user can click this button to switch back to the geolocated country like WCCOM NUX. 🙂

Reply:
I didn't realise it was clickable 😆 
Fixed in https://github.com/woocommerce/woocommerce/pull/38412/commits/4c3e694332bde603017409c84ae2966f1785aa94